operations/host_spawn.go

```diff
@@ -89,29 +95,42 @@ func hostCreate() cli.Command {
 			client := conf.setupRestCommunicator(ctx)
 			defer client.Close()
 
-			var script string
+			spawnRequest := &restModel.HostRequestOptions{}
+			tags := []host.Tag 
```

Comment:
No reason to initialize this outside of the if/else since each handles it separately.

Reply:
thanks!

Reply:
Ah I remember now why I did that. The linter made me define it separately because otherwise err shadows the above definition. Since I need to define it separately anyhow, and it will end up being used one way or another (so there's no extra overhead), it feels cleaner to define it once instead of defining it twice.